File: VB/DxSample/Editors/SimpleGroupBorderPainter.vb
Imports DevExpress.Skins
Imports DevExpress.Utils.Drawing

Namespace DxSample.Editors
	Public Class SimpleGroupBorderPainter
		Inherits SkinGroupBorderPainter

		Public Sub New(ByVal groupPainter As SkinGroupObjectPainter, ByVal provider As ISkinProvider)
			MyBase.New(groupPainter, provider)
		End Sub

		Protected Overrides Function GetPanelSkinElement(ByVal info As GroupObjectInfoArgs) As SkinElement
			Return CommonSkins.GetSkin(Me.Provider)(CommonSkins.SkinGroupPanel)
		End Function
	End Class
End Namespace

File: CS/DxSample/Editors/SimpleGroupBorderPainter.cs
using DevExpress.Skins;
using DevExpress.Utils.Drawing;

namespace DxSample.Editors {
    public class SimpleGroupBorderPainter :SkinGroupBorderPainter {
        public SimpleGroupBorderPainter(SkinGroupObjectPainter groupPainter, ISkinProvider provider) : base(groupPainter, provider) { }

        protected override SkinElement GetPanelSkinElement(GroupObjectInfoArgs info) {
            return CommonSkins.GetSkin(this.Provider)[CommonSkins.SkinGroupPanel];
        }
    }
}
